pkg/js-search/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "js-search",
+  "version": "2.0.0",
+  "description": "JS Search is an efficient, client-side search library for JavaScript and JSON objects",
+  "main": "./source/index.ts",
+  "module": "./source/index.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/umd/js-search.js",
+      "import": "./source/index.ts"
+    }
+  },
+  "files": [
+    "dist",
+    "source"
+  ],
+  "devDependencies": {
+    "@babel/core": "7.x",
+    "@babel/preset-env": "^7.8.7",
+    "babel-jest": "^25.1.0",
+    "jest": "^27.5.1",
+    "prettier": "^2.6.2",
```

Comment:
I don't think prettier is necessary here. If you want format on save I believe you need to have the prettier installed globally (+ the extension). And if you don't use it, and just want to format pre commit, that will be done through pre-commit (with prettier installed in `common/autoinstallers/pre-commit` for that specific use case.

Reply:
Sry: nitpicky minimalism

Reply:
I'm not 100% convinced: if we rely on a system installed `prettier` we can't pin a specific version, so it might unpredictably introduce changes if different people have a different `prettier` version.

Anyway, I removed the dependency for now.

Reply:
Yeah, but in [autoinstaller](https://github.com/librocco/librocco/blob/main/common/autoinstallers/lint-staged/package.json) we're specifying the right version (and that is the one used to run pre-commit). Locally installed (well globally but on the developer's machine) is only for development comvenience.

Reply:
Yeah, but in [autoinstaller](https://github.com/librocco/librocco/blob/main/common/autoinstallers/lint-staged/package.json) we're specifying the right version (and that is the one used to run pre-commit). Locally installed (well globally but on the developer's machine) is only for development comvenience.